libse/SubtitleFormats/DvdSubtitleSystem.cs

```diff
@@ -64,25 +55,27 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
             _errorCount = 0;
             foreach (string line in lines)
             {
-                if (RegexTimeCodes.IsMatch(line))
+                // line must contain atleast 24 characters (time-code)...
+                if (line.Length < 24)

```

Comment:
Regex pattern is equal to 24 to there is no need to force regex matching shorted text (for example others subtitle formats...)